mlflow/models/evaluation/base.py

```diff
@@ -1262,6 +1262,15 @@ def evaluate(
           specified, unless the ``evaluator_config`` option **log_model_explainability** is
           explicitly set to ``True``.
 
+    .. note::
+
+        To compute and log a full set of metrics and artifacts supported by the default
+        evaluator, use the following command to install required extra dependencies:
+
+        .. code-block:: bash
+
+            pip install 'mlflow[evaluate]'
```

Comment:
I think this is better than `question-answering requires a, b, and c. text-summarization requires x, y, and z`.

Reply:
Agreed! The current solution is hard for users to follow